etl-federation/README.md

```diff
@@ -2,9 +2,14 @@
 
 These are the steps to enable ETL federation with Kubecost.
 
-This is an efficient method to implement multi-cluster Kubecost while using existing prometheus installations. Contact us for help customizing settings.
+This is an efficient method to implement multi-cluster Kubecost while using existing prometheus installations. The example in this directory uses the Kubecost Prometheus server. See the [existing-prometheus](./existing-prometheus/) directory for additional configuration required.
+
+Contact us for help customizing settings.
```

Comment:
I recommend to add a hyperlink here:
Example:

[Contact us](https://www.kubecost.com/contact) for help customizing settings.